partiql-tests-data/fail/syntax/query/select/select.ion

```diff
@@ -61,3 +61,79 @@
         result: SyntaxFail
     },
 }
+
+'select-star'::[
```

Comment:
Does it makes sense to put these tests under `partiql` namespace to re-iterate the fact that they are not necessarily SQL conformed.

Reply:
Yeah, it would probably make some sense to indicate that these tests are not necessarily SQL-conformant. I'll wrap these tests in a `partiql` namespace and add a comment in the next commit.